readme.md

```diff
@@ -16,9 +16,9 @@ $ npm install --save screenshot-stream
 const fs = require('fs');
 const screenshot = require('screenshot-stream');
 
-const stream = screenshot('http://google.com', '1024x768', {crop: true});
-
-stream.pipe(fs.createWriteStream('google.com-1024x768.png'));
+screenshot('http://google.com', '1024x768', {crop: true}).then(data => {
```

Comment:
I wonder if we should just make the `size` argument part of the options object with a good default.

Reply:
Yup, let's do that.

Reply:
👌